ajax/libs/rsuite/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rsuite",
+  "filename": "rsuite.min.js",
+  "version": "3.0.2",
+  "description": "A suite of react components",
+  "homepage": "https://rsuitejs.com/",
+  "keywords": [
+    "react",
+    "rsuite",
+    "component",
+    "react-component"
+  ],
+  "author": "Simon Guo <simonguo.2009@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:rsuite/rsuite.git"
```

Comment:
Let's revise to `git://github.com/rsuite/rsuite.git`
We provide the public URL to let people access the link.
People need to add the ssh public key to GitHub, then he(she) can use the current URL.
Many thanks.